### ISSUE TITLE
[BUG]: Ankidroid treats type/queue of card differently from Anki(Desktop) and it's affecting scheduler

### ISSUE THREAD

**Issue by kaCVanime** [ankidroid/Anki-Android#17148]
### Checked for duplicates?

- [X] This issue is not a duplicate

### Does it also happen in the desktop version?

- [X] This bug does not occur in the latest version of Anki Desktop

### What are the steps to reproduce this bug?

1. open Anki desktop, create a new deck, then add a card to it. close anki.
2. open collection.anki2 db file, in card table, for the newly added card, modify its field `type` to 3 and field `queue` to 1, save changes.
3. open anki again, go to the review page of the card, check the days of Good/Easy button at the bottom bar.
4. sync.  
---
5. open Ankidroid, sync.
6. go to the review page of the card added, check the days of Good/Easy button at the bottom bar to see if these are same as in Anki desktop.
---
7. on Anki desktop, modify the FSRS params in deck options of the card, save. Then go to review page of the card to see if there is a change of the days.
8. on Ankidroid, modify the FSRS params in deck options of the card, save. Then go to review page of the card to see if there is a change of the days.

### Expected behaviour

What happened:
a. the days of Good/Easy button in Ankidroid (step 6) is different from those in Anki desktop (step 3).
b. after modifying the FSRS params, the days of Good/Easy button change in Anki desktop, but remains unchange in Ankidroid.

What should have happened:
a. the days of Good/Easy button in Ankidroid (step 6) should be same as those in Anki desktop (step 3).
b. after modifying the FSRS params, the days of Good/Easy button should change.

### Debug info

```text
AnkiDroid Version = 2.18.4 (f867da96b8304edaebd0361326316bc7514f8cde)

Backend Version = 0.1.38-anki24.04.1 (24.04.1 ccd9ca1a8309b80bcb50ddc5d99c7ce63440bce9)

Android Version = 14 (SDK 34)

ProductFlavor = play

Manufacturer = OnePlus

Model = PJD110

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 14; PJD110 Build/UKQ1.230924.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/126.0.6478.71 Mobile Safari/537.36

ACRA UUID = e33baacc-459c-473a-8547-6f4a2cd870cf

FSRS Enabled = true

Crash Reports Enabled = true
```

### (Optional) Anything else you want to share?

Anki debug info
```
Anki 24.06.3 (d678e393)  (ao)
Python 3.9.18 Qt 6.6.2 PyQt 6.6.1
Platform: Windows-10-10.0.22631

===Add-ons (active)===
(add-on provided name [Add-on folder, installed at, version, is config changed])
'' ['fill-the-blanks', 0, 'None', '']
'' ['myaddon', 0, 'None', '']
'' ['suspend-by-tags', 0, 'None', '']
AnkiConnect ['2055492159', 2024-07-26T12:55, 'None', '']
AnkiWebView Inspector ['31746032', 2023-06-28T03:26, 'None', '']
FSRS Helper Postpone  Advance  Load Balance  Easy Days  Disperse Siblings ['759844606', 2024-09-25T14:18, 'None', mod]
FastWordQuery-Qt6 ['103636257', 2024-04-07T02:28, 'None', '']
Remove Duplicate Cards Enhanced ['Remove-Duplicate-Cards-Enhanced', 2020-04-25T14:55, 'None', '']
Remove Duplicate Cards ['95590040', 2024-09-10T22:46, 'None', '']
Review Heatmap ['1771074083', 2022-06-30T09:43, 'None', '']

===IDs of active AnkiWeb add-ons===
103636257 1771074083 2055492159 31746032 759844606 95590040

===Add-ons (inactive)===
(add-on provided name [Add-on folder, installed at, version, is config changed])

```

### Research

- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by user1823** [comment]
> open collection.anki2 db file, in card table, for the newly added card, modify its field type to 3 and field queue to 1, save changes.

AFAIK, such changes are not synced during normal syncs. You need to force a one-way sync to get those changes on other devices.

> after modifying the FSRS params, the days of Good/Easy button change in Anki desktop, but remains unchange in Ankidroid.

This looks like a bug that was fixed in Anki 24.06.3. The changes will be incorporated in AnkiDroid 2.19

**Comment by david-allison** [comment]
Please confirm whether this is still an issue in 2.19

**Comment by kaCVanime** [comment]
@user1823 @david-allison 
Thanks for the quick reply. I've confirmed this issue is solved in 2.19.

**Comment by david-allison** [comment]
Thanks for the quick reply!!
